prometheus/README.md

```diff
@@ -0,0 +1,61 @@
+# Prometheus
+
+This sample app shows how to ingest data from [Prometheus](https://prometheus.io/) into [Deephaven](https://deephaven.io/).
```

Comment:
worth explaining what Prometheus is?

Reply:
`Prometheus is an open-source systems monitoring and alerting toolkit that collects and stores its metrics as time series data` how does that sound? I took it from https://prometheus.io/docs/introduction/overview/

Reply:
Perfect!